docs/tutorials/t-05.md

```diff
@@ -0,0 +1,124 @@
+- JavaScript package manager - either [YARN](https://classic.yarnpkg.com/en/docs/install) > v3.1.0 or NPM which comes with NodeJS.
+- IDE, e.g. [VSCode](https://code.visualstudio.com/download).
+- Setup the accounts on [`Algosigner`](https://github.com/PureStake/algosigner), [`Wallet Connect`](https://walletconnect.com/) wallet and [`MyAlgo Wallet`](https://wallet.myalgo.com/). It could be MainNet, TestNet or private net.
+
+## Steps
+
+You can create a wallet widget to connect to different wallets using our `Wallet Widget`. To add it, follow the steps mentioned in this [tutorial]()
+
+### MultiSignature Transaction
+
+To create a msig transaction, you can follow steps from the offical docs of Algorand Foundation to create a [Multisignature address](https://developer.algorand.org/docs/get-details/accounts/create/#multisignature) and [Transaction](https://developer.algorand.org/docs/sdks/javascript/) using it.
+Here is a glimpse of the creating the multisig transaction:
+
+```js
+const mparams = {
+	version: 1,
+	threshold: 2,
+	addrs: [account1.addr, account2.addr, account3.addr],
+};
+
+let multsigaddr = algosdk.multisigAddress(mparams);
+let msigTxn = algosdk.makePaymentTxnWithSuggestedParamsFromObject({
+	from: multsigaddr,
+	to: receiver,
+	amount: amount,
+	note: note,
+	suggestedParams: params,
+});
+```
+
+After creating the transaction, you can sign it using different wallets.
+
+1. AlgoSigner<br/>
+We use `connectAlgoSigner()` method to connect the AlgoSigner wallet. Next, we create an instance of the `WebMode` class by passing `AlgoSigner` and the Chain Name i.e the network name. The `web` instance will allow us to sign and send transactions to the blockchain.
+
+```js
+const web = new WebMode(AlgoSigner, CHAIN_NAME);
+```
+
+Webmode method `signMsigTxn` expects txn in [`EncodedSignedTransaction`](https://algorand.github.io/js-algorand-sdk/interfaces/EncodedSignedTransaction.html) type. To convert the above created msig transaction, following code is used:
+
+```js
+let subsig = [];
+for (const address of mparams.addrs) {
+	subsig.push({ pk: decodeAddress(address).publicKey });
+}
+const msigEncodedTxn = {
+	msig: { v: mparams.version, thr: mparams.threshold, subsig: subsig },
+	txn: msigTxn.get_obj_for_encoding(),
+};
+```
+
+To sign the txn:
+
+```js
+const signedTxnResponse = await web.signMsigTxn([msigEncodedTxn]);
+```
+
+The following screen will popup for you to sign the txn:
+<br/>
+<img src="./t-05/assets/algosigner.png" height="500" title="Sign Msig Txn" />
+
+After signing, the response is the object containing blob key, which we can use further to send the signed transaction to the network.
+To send the above txn we need to first convert it to base64:
+
+```js
+const base64Txn = Buffer.from(signedTxnResponse.blob).toString("base64");
+const sendTxnResponse = await webmode.sendAndWait(base64Txn);
+```
+
+The response includes the confirmed transaction response along with txID.
```

Comment:
we repeat response twice. We should rephrase it

Reply:
oh yes, will update